composer.json

```diff
@@ -31,10 +31,14 @@
     "require-dev": {
         "silverstripe/documentation-lint": "^1",
         "silverstripe/recipe-testing": "^3",
+        "silverstripe/behat-extension": "^5.6.1",
```

Comment:
This includes the fix for the workaround https://github.com/silverstripe/silverstripe-behat-extension/pull/308